### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by 21stSentry** [apohl79/audiogridder#1384] [General]
Haven't seen any responses or updates from apohl in a while.

**Comment by zecevic** [comment]
I have same question

**Reply by apohl79** [reply]
It's not dead. I have no capacity at the moment, but this will change.

**Reply by zecevic** [reply]
Thank you. That is excellent news. I have been using it for a while now, and I have nothing but praises

**Reply by kcoul** [reply]
Will be finally working on a new feature set that I was planning for some time now, in the coming months (to optionally be able to use the built-in MIDI and Audio I/O of servers)

**Reply by zecevic** [reply]
Great

**Reply by 21stSentry** [reply]
Great news!

**Reply by Deltager** [reply]
Nice to hear. Maybe support for GPU as well ? Would that be possible ?

**Reply by kcoul** [reply]
@Deltager it is technically possible thanks to release of GPUAudio SDK but it would really be up to the plugin vendor to support it, not AudioGridder. I think with a plugin that already had support for GPU-based processing, it would already work, but they are just starting to appear now that the SDK is public.

**Reply by StillHollow** [reply]
Starting to see real interest in the Audio Gridder platform from LUNA DAW users who have Intel processors. UA devs are coding the latest builds geared towards M1 or newer processors and those of us on Intel, especially i7 processors are hitting the wall quick even after a very few tracks/plugins. Running Audio Gridder even in local mode dramatically improves performance. 

Please continue to develop this, it is an amazing thing. I will donate at the website if you say development is continuing, I will spread the word over on the UA forum as well or perhaps you could drop in and give an update. Some frustration from those of us on Intel and older machines and although development always moves forward many of us who have PC's which have no trouble running large sessions in Pro Tool or Logic don't want to have to buy a new machine.

**Reply by mabian69** [reply]
Any news?

**Reply by kwalker99** [reply]
> Will be finally working on a new feature set that I was planning for some time now, in the coming months (to optionally be able to use the built-in MIDI and Audio I/O of servers)

Did this ever happen? Would love to be able to use my RME MADI I/O but with the GUI on my mac daw etc

**Reply by kcoul** [reply]
@kwalker99 I think I will be able to have another go at this from around June, will report back once it's in development again!

**Comment by Deltager** [comment]
I currently using AnalogX’s Genesis. I know they are working towards GPUAudio support, so maybe there will be a solution soon.
These guys are very dedicated, and working hard to make Genesis ready for GPU-processing. I guess we just have to wait and see ...

**Reply by kcoul** [reply]
I think it would be brilliant to have separate boxes optimized for running absurd amounts of compute-intensive processing like convolution reverb on the GPU, so I have a personal interest in getting that working. At that scale, it would make sense to also enable routing audio between machines over protocols like AVB, Dante and MADI, something I would be interested in looking into as a follow-up feature.

**Comment by poppagreg** [comment]
No new releases or much activity since 2023. Can we get some other developers working on this project? I am sure people would be willing to support it.

**Reply by kcoul** [reply]
I'm going to try and get some attention on this project by making it the topic of a talk at Audio Developer Conference, but that won't be until November. 

In the meantime, I think the feature I'm trying to add should be really useful for some setups (allowing servers to use their own audio and MIDI I/O instead of the client's)

**Reply by AnietyDev** [reply]
Be the change you want, the source code is public. Fix the major issues you have needed fixed since 2023 or since you say you are willing to support there are places you can find developers. Maybe pay them to fix what you have been waiting on? 

Or support the current dev for the work that they have done up to this point that has you coming here to comment.

**Reply by poppagreg** [reply]
> I'm going to try and get some attention on this project by making it the topic of a talk at Audio Developer Conference, but that won't be until November.
> 
> In the meantime, I think the feature I'm trying to add should be really useful for some setups (allowing servers to use their own audio and MIDI I/O instead of the client's)

Sounds good, the waiting game. Something is better than nothing.

**Reply by poppagreg** [reply]
> Be the change you want, the source code is public. Fix the major issues you have needed fixed since 2023 or since you say you are willing to support there are places you can find developers. Maybe pay them to fix what you have been waiting on?
> 
> Or support the current dev for the work that they have done up to this point that has you coming here to comment.

What planet are you from?

**Reply by JulienRik** [reply]
in the meantime, what yall think baout vienna ensemble pro 7? sry for mention it, im quit a opensource dude too, but also doin live n studio. like things running smooth.

**Comment by vincentark** [comment]
Vienna is great, I use v5 sometimes, but you don't see the plugins on your main computer, only on the slaves (unless they changed that). 
There is a demo available!

**Comment by paolocampitelli** [comment]
up.......

**Comment by Deltager** [comment]
Great to hear !Thanks 😎Sendt fra min iPhone5. mai 2026 kl. 18:10 skrev kcoul ***@***.***>:﻿
@kwalker99 I think I will be able to have another go at this from around June, will report back once it's in development again!

**Comment by cyberishh** [comment]
No news, good news?
